scene/resources/multimesh.cpp

```diff
@@ -275,10 +333,13 @@ void MultiMesh::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_instance_color", "instance"), &MultiMesh::get_instance_color);
 	ClassDB::bind_method(D_METHOD("set_instance_custom_data", "instance", "custom_data"), &MultiMesh::set_instance_custom_data);
 	ClassDB::bind_method(D_METHOD("get_instance_custom_data", "instance"), &MultiMesh::get_instance_custom_data);
+	ClassDB::bind_method(D_METHOD("set_as_bulk_array"), &MultiMesh::set_as_bulk_array);
```

Comment:
You should give a name to the argument, otherwise it shows as `arg0` in the doc. Should be `D_METHOD("set_as_bulk_array", "array")` or similar.

Reply:
Oops, I usually don't forget that :)